code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/components/ImageOnlyCard.kt

```diff
@@ -62,14 +62,14 @@ fun ImageOnlyCard(
             }
         ) {
             Box {
-                AepAsyncImage(
+                AepAsyncImageComposable(
```

Comment:
I prefer what you added previously without`*Composable` suffix for these internal components 😄.